packages/driver/cypress/fixtures/a11y/slider.html

```diff
@@ -0,0 +1,231 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>A11y Test Bed - Range Slider</title>
+  <style> 
+    .styled-test-slider-container {
+      width: 129px;
+      height: 8px;
+      background-color: #0075ff;
+      display: inline-block;
+      margin: 2px;
+      border-radius: 8px;
+      background-size: 125px 8px;
+    }
+
+    .styled-test-slider {
+      display: inline-block;
+      width: 16px;
+      height: 16px;
+      border-radius: 50%;
+      background-color: #0075ff;
+      position: relative;
+      left: calc(50% - 10px);
+      top: -50%;
+      z-index: 1;
+    }
+
+    .styled-label {
+      display: block;
+    }
+  </style>
+</head>
+<body>
+  <div>
+    <!-- Semantic Slider -->
+    <label
+      for="test-a11y-semantic-slider"
+      class="styled-label"
+    >
+      Semantic Points
+    </label>
+    <input
+      type="range"
+      min="0"
+      max="10"
+      value="5"
+      id="test-a11y-semantic-slider"
+    />
+
+    <!-- Styled Slider -->
+    <div>
+      <label
+        id="test-a11y-styled-slider-label"
+        class="styled-label"
+      >
+        Styled Points
+      </label>
+      <div
+        id="test-a11y-styled-slider-container"
+        class="styled-test-slider-container"
+      >
+          <div
+            id="test-a11y-styled-slider"
+            class="styled-test-slider"
+          />
+      </div>
+    </div>
+
+    <!-- Role Slider -->
+    <div>
+      <label
+        id="test-a11y-role-slider-label"
+        class="styled-label"
+      >
+        Role Points
+      </label>
+      <div
+        role="slider"
+        id="test-a11y-role-slider-container"
+        aria-orientation="horizontal"
+        tabindex="0"
+        aria-valuemin="0"
```

Comment:
Interesting, I had no idea you could do an accessible slider using role - I've only used `range`, which I guess isn't acesssible?

Reply:
I'm not sure if it _isn't_ accessible, but `range` is the superclass for the type, so the guidance is to use a more specific subclass.
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/range_role